vm/interpreter/src/default_runtime.rs

```diff
@@ -985,26 +985,35 @@ where
     ) -> Result<HashMap<Address, Vec<bool>>, Box<dyn StdError>> {
         // Gas charged for batch verify in actor
 
-        let out = vis
-            .par_iter()
-            .map(|(&addr, seals)| {
-                let results = seals
-                    .par_iter()
-                    .map(|s| {
-                        if let Err(err) = V::verify_seal(s) {
-                            debug!(
-                                "seal verify in batch failed (miner: {}) (err: {})",
-                                addr, err
-                            );
-                            false
-                        } else {
-                            true
-                        }
-                    })
-                    .collect();
-                (addr, results)
-            })
-            .collect();
+        let cpus = num_cpus::get();
```

Comment:
Consider using `lazy_static` to evaluate this function once and set a constant, module level variable

Reply:
or [once_cell](https://github.com/matklad/once_cell)